Stripe/STPAPIClient.m

```diff
@@ -260,9 +260,42 @@ - (void)createTokenWithPersonalIDNumber:(NSString *)pii completion:(__nullable S
 
 @implementation STPAPIClient (Upload)
 
+- (NSData *)dataForUploadedImage:(UIImage *)image
```

Comment:
JPEG file size is largely dependent on how complicated the image you are encoding is, which is hard to quantify exactly.